lib/screens/home/cocktail_app_home_menu_items.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:boilerplate/styles/images.dart';
+
+enum CocktailAppHomeMenuItems
+{
+  category(name: 'Category'),
+  typeOfGlass(name: 'Type of glass'),
+  ingredient(name: 'Ingredient'),
+  alcoholic(name: 'Alcoholic (or not)');
+
+  final String name;
+  const CocktailAppHomeMenuItems({required this.name});
+}
+
+extension HomeItemsExt
```

Comment:
Use complete name when naming variables, extension or functions
```suggestion
extension HomeMenuItemsExtension
```

Reply:
Suggested changes already pushed. Thank you for your comments and help @keylinwu-dnamic !